File: java/questao5.java
import java.util.Scanner;

public class questao5 {
    public static void main(String args[]){
        Scanner scan = new Scanner(System.in);
        System.out.print("Digite a primeira nota: ");
        float nota1 = scan.nextFloat();
        System.out.print("Digite o peso da primeira nota: ");
        float peso1 = scan.nextFloat();
        
        System.out.print("Digite a segunda nota: ");
        float nota2 = scan.nextFloat();
        System.out.print("Digite o peso da terceira nota: ");
        float peso2 = scan.nextFloat();

        System.out.print("Digite a terceira nota: ");
        float nota3 = scan.nextFloat();
        System.out.print("Digite o peso da terceira nota: ");
        float peso3 = scan.nextFloat();

        float mediaponderada = (nota1*peso1 + nota2 *peso2 +nota3*peso3)/peso1+peso2+peso3;

        System.out.print("A média ponderada das notas fornecidas é: ");
        System.out.println(mediaponderada);





    }
}

File: c/questao5.c
#include <stdio.h>
#include <stdlib.h>

int main(){

    float nota1, nota2, nota3,somanota;
    printf("Digite as tres notas: ");
    scanf("%f%f%f",&nota1,&nota2,&nota3);
    somanota = (nota1+nota2+nota3)/3;
    printf("A media geral das notas e: %.2f", somanota);
}